packages/mui-base/src/Dialog/Title/DialogTitle.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { createRenderer } from '@mui/internal-test-utils';
+import * as Dialog from '@base_ui/react/Dialog';
+import { describeConformance } from '../../../test/describeConformance';
+
+describe('<Dialog.Title />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<Dialog.Title />, () => ({
+    refInstanceof: window.HTMLHeadingElement,
+    render: (node) => {
+      return render(
+        <Dialog.Root open>
+          <Dialog.Popup animated={false}>{node}</Dialog.Popup>
```

Comment:
I placed `animated` on the `Root` in Tooltip, as that's where the data is required in the flow

Reply:
> that's where the data is required in the flow

What do you mean? Does the root need to know if subcomponents are animated in your case?

Reply:
Yes, the Root is what contains the interactions logic that changes the `open` state, as it's shared between the trigger and popup. So the prop must live in the Root for Tooltip/Popover.

Reply:
OK, done here as well